### DISCUSSION TITLE
Bob Morane exists but not Lee Enfield in search

### DISCUSSION THREAD

**Discussion by MugUK** [atari-legend/atari-legend#197] [Q&A] [answered]
If we have a review in an English magazine of a Bob Morane classic (https://www.atarilegend.com/games/744), but renamed to Lee Enfield (https://www.atarilegend.com/games/release/9483), I can see that both titles exist.

However when I try to add the Lee Enfield game as a review in the first ST Action magazine that I found on archive.org - it doesn't exist.

Am I doing something wrong?

**Comment by nguillaumin** [comment, marked as answer]
Ah yes, it'a because Lee Enfield is a release of the Bob Morane game. It just had a special name when released it the UK. Thus the correct thing here is too link the Bob Morane game. You can put the Lee Enfield title in the meantime index title as an extra info.

**Comment by MugUK** [comment]
They were all ports of crap Amstrad games, but with some sampled sound extra. I'll go and fix them as you've suggested.
